pkg/commands/oscommands/cmd_obj.go

```diff
@@ -104,7 +107,11 @@ func (self *CmdObj) GetCmd() *exec.Cmd {
 }
 
 func (self *CmdObj) ToString() string {
-	return self.cmdStr
+	return strings.Join(self.ToArgs(), " ")
```

Comment:
I think it would be good to quote the args if needed before joining them. Maybe something simple like adding double quotes if the arg contains a space could do. That way it is more likely to work if you copy and paste a command from the log.

Reply:
Good idea